packages/web/src/generators/application/files/app-vite/src/app/app.element.spec.ts__tmpl__

```diff
@@ -0,0 +1,21 @@
+import { AppElement } from './app.element';
```

Comment:
Can you add a flag for this file to be generated? When using Vite i really prefer using in file testing. 

I hope you dont mind the comment.

Reply:
The testing configurations will happen in a later PR, so I will leave this as is for now, but will keep this in mind when we're doing the test generators! Thank you very much for the feedback!